Sources/SimpleSalesforceSDK/Services/WebService.swift

```diff
@@ -79,19 +74,24 @@ class WebService {
   // MARK: - Utilities
 
   static func makeRequest(_ request: URLRequest, ignore401: Bool = false) async throws -> (Data, Int) {
-    let (data, response) = try! await URLSession.shared.data(for: request)
+    let (data, response) = try await URLSession.shared.data(for: request)
+
     guard let httpResponse = response as? HTTPURLResponse else {
       throw SSSDKError.notOk
     }
 
     let statusCode = httpResponse.statusCode
     guard statusCode >= 200 && statusCode < 299 || (ignore401 && statusCode == 401) else {
-      if let jsonString = String(data: data, encoding: .utf8) {
-        print(jsonString)
```

Comment:
since you removed the printing of the error, how would the caller know if it is not a `authRefreshTokenExpiredError` error?

Reply:
Maybe you can include text in the `SSSDKError.notOk` error or just print it like before